src/components/BtcTransactionList.vue

```diff
@@ -62,7 +62,7 @@
         </RecycleScroller>
 
         <div v-else-if="!searchString" class="empty-state flex-column">
-            <h2 class="nq-h1">{{ $t('Your transactions will appear here') }}</h2>
+            <h2 class="nq-h1">{{ $t('Nothing here yet') }}</h2>
```

Comment:
I am not sure if this PR should include this change. In Figma there are a few different texts, should I update them?